### PR TITLE
Submission

### PR DESCRIPTION
**Pull request by vasilisa-iatckova**
Ok this should be correct?

### CODE REVIEW

File: assignment.md [hunk 1]
```diff
@@ -11,63 +11,113 @@ If the version control concepts or `git` commands are still confusing to you, co
...
     ```
     This is also a good opportunity to turn your dotfiles folder into a version controlled repo (Use `git init` in your dotfiles folder), make it private if you have sensitive `ssh` configurations there (IP Addresses), else feel free to make it public. (This is for your own benefit, not part of the assignment grading)
 
  8. You can define global ignore patterns in `~/.gitignore_global` after running `git config --global core.excludesfile ~/.gitignore_global`. Do this, and set up your global `gitignore` file to ignore OS-specific or editor-specific temporary files, like `.DS_Store`, or `.vscode`. Feel free to Google around for common gitignore entries for Python projects or other projects of your interest.
 
     ```bash
-    # Your .gitignore_global entries here
+    .DS_Store
+    .DS_Store?
+    Thumbs.db
```
Comment (axiezai) [inline assignment.md:120]: For python projects, it's useful to add `__pycache__`, `.ipynb_checkpoints`, and `.vscode`.

File: assignment.md [hunk 2]
```diff
@@ -11,63 +11,113 @@ If the version control concepts or `git` commands are still confusing to you, co
...
 
  6. One common mistake when learning `git` is to commit large files that should not be managed by `git` or adding sensitive information like security keypairs for Amazon Cloud Services. Navigate back to your assignment repository and try adding a random text file to the repository (use `touch`), making some modifications and commits to your repository, and deleting that file with `rm`. But deleting with `rm` will not delete that file's recorded git history, therefore we need to do a little more. You may want to look at [this](https://help.github.com/articles/removing-sensitive-data-from-a-repository/))
 
-    > Perform the commits and deletion, this question will be graded based on your commit history on GitHub. I should see commits for when you added your file and made modifications, and I should see a commit for when you deleted the files. 
+    > I accidentally did this on main the first time around, but we fixed it together later.
```
Comment (axiezai) [inline assignment.md:105]: I think when we did this together, we only used `git rm --cached` to make sure there was nothing wrong with ur git repo and didn't include the entire command for filtering and replacing text. 

So your "iamlisa" is still visible in your commit history files, it's my bad for taking the easier route when we did this together, but for future reference... remove using the full command to erase that text from your history.

**Review by axiezai — APPROVED** [review]
Good job, 100%. See my comments about the password... my bad

File: assignment.md (hunk 1 above)
Comment (vasilisa-iatckova) [reply assignment.md:120]: Thanks!!!

File: assignment.md (hunk 2 above)
Comment (vasilisa-iatckova) [reply assignment.md:105]: I see! I usually don't ever put anything sensitive on, but I'll keep that in mind! I still don't know how to look at whether it's visible though.

**Pull request merged**